configurePipelineFiles.sh

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env bash
+
+CURRENT_DIR=`dirname $0`
+
+# Colors and styling for the command line
+DIM='\033[2m'
+BOLD='\033[1m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+WHITE='\033[1;37m'
+CHECK='\xE2\x9C\x94'
+BLACK_ON_GREY='\033[30;47m'
+RESET='\033[0m'
+
+WELCOME_MSG="
+This is an Appian-provided macOS command line tool that will help configure a sample pipeline for use"
+
+ADM_MSG="
+┌------------------- ADM Configuration ---------------------┐
+|    Let's configure the ADM tools for use (AIM and AVM).   |
+|    The prerequisite is that you must have a AVM built     |
+|    application residing in the appian/applications folder |
+└-----------------------------------------------------------┘
```

Comment:
How about something like "In addition to the configurations that are being set up now, you'll need to add and AVM build application to appian/applications folder"
 saying that the application is a prerequisite might be slightly ambiguous as to whether they need that before they proceed with the script...